### PR TITLE
fix: Fix licences for libs

### PR DESCRIPTION
**Pull request by hernandazevedozup**

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/beagle-defaults/build.gradle [hunk 1]
```diff
@@ -56,23 +56,21 @@ dependencies {
 
     implementation "br.com.zup.beagle:android:$beagle_version"
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9'
```
Comment (botbeaglezup) [inline android/beagle-defaults/build.gradle:58]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults/build.gradle#L58"></span>A newer version of org.jetbrains.kotlinx:kotlinx-coroutines-android than 1.3.9 is available: 1.4.2</td>
    </tr>
  </tbody>
</table>

File: android/beagle-defaults/build.gradle [hunk 2]
```diff
@@ -56,23 +56,21 @@ dependencies {
...
-    implementation "org.junit.vintage:junit-vintage-engine:5.7.0"
-    testImplementation "io.mockk:mockk:1.10.2"
-    testImplementation project (":android-test")
 
-    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    testImplementation "br.com.zup.beagle:android-test:$beagle_version"
     testImplementation "org.robolectric:robolectric:4.4"
-    testImplementation 'junit:junit:4.+'
+    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.3.9"
```
Comment (botbeaglezup) [inline android/beagle-defaults/build.gradle:62]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults/build.gradle#L62"></span>A newer version of org.jetbrains.kotlinx:kotlinx-coroutines-test than 1.3.9 is available: 1.4.2</td>
    </tr>
  </tbody>
</table>

File: android/beagle-defaults/build.gradle [hunk 3]
```diff
@@ -56,23 +56,21 @@ dependencies {
...
+    testImplementation "org.junit.jupiter:junit-jupiter-engine:5.7.0"
     testImplementation "org.junit.vintage:junit-vintage-engine:5.7.0"
     testImplementation "androidx.test:core:1.4.0-alpha04"
```
Comment (botbeaglezup) [inline android/beagle-defaults/build.gradle:65]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults/build.gradle#L65"></span>A newer version of androidx.test:core than 1.4.0-alpha04 is available: 1.4.0-beta01</td>
    </tr>
  </tbody>
</table>

File: android/beagle-defaults/build.gradle [hunk 4]
```diff
@@ -56,23 +56,21 @@ dependencies {
...
     testImplementation "androidx.test:runner:1.4.0-alpha04"
```
Comment (botbeaglezup) [inline android/beagle-defaults/build.gradle:66]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults/build.gradle#L66"></span>A newer version of androidx.test:runner than 1.4.0-alpha04 is available: 1.4.0-beta01</td>
    </tr>
  </tbody>
</table>

File: android/beagle-defaults/build.gradle [hunk 5]
```diff
@@ -56,23 +56,21 @@ dependencies {
...
+    testImplementation "junit:junit:4.13.2"
+    testImplementation "org.junit.jupiter:junit-jupiter-api:5.7.0"
+    testImplementation "io.mockk:mockk:1.10.2"
```
Comment (botbeaglezup) [inline android/beagle-defaults/build.gradle:69]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults/build.gradle#L69"></span>A newer version of io.mockk:mockk than 1.10.2 is available: 1.10.6</td>
    </tr>
  </tbody>
</table>

File: android/beagle-defaults/build.gradle [hunk 6]
```diff
@@ -56,23 +56,21 @@ dependencies {
...
+
     androidTestImplementation "androidx.test.ext:junit:1.1.3-alpha04"
```
Comment (botbeaglezup) [inline android/beagle-defaults/build.gradle:71]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults/build.gradle#L71"></span>A newer version of androidx.test.ext:junit than 1.1.3-alpha04 is available: 1.1.3-beta01</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/build.gradle [hunk 7]
```diff
@@ -63,27 +63,25 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
```
Comment (botbeaglezup) [inline android/beagle-scaffold/build.gradle:63]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold/build.gradle#L63"></span>A newer version of androidx.appcompat:appcompat than 1.2.0 is available: 1.3.0</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/build.gradle [hunk 8]
```diff
@@ -63,27 +63,25 @@ dependencies {
...
     implementation 'com.google.android.material:material:1.3.0'
 
-
     implementation "br.com.zup.beagle:android:$beagle_version"
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9'
```
Comment (botbeaglezup) [inline android/beagle-scaffold/build.gradle:67]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold/build.gradle#L67"></span>A newer version of org.jetbrains.kotlinx:kotlinx-coroutines-android than 1.3.9 is available: 1.4.2</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/build.gradle [hunk 9]
```diff
@@ -63,27 +63,25 @@ dependencies {
...
-    implementation "org.junit.vintage:junit-vintage-engine:5.7.0"
-    testImplementation "io.mockk:mockk:1.10.2"
-    testImplementation project (":android-test")
-
-    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    testImplementation "br.com.zup.beagle:android-test:$beagle_version"
     testImplementation "org.robolectric:robolectric:4.4"
-    testImplementation 'junit:junit:4.+'
+    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.3.9"
```
Comment (botbeaglezup) [inline android/beagle-scaffold/build.gradle:72]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold/build.gradle#L72"></span>A newer version of org.jetbrains.kotlinx:kotlinx-coroutines-test than 1.3.9 is available: 1.4.2</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/build.gradle [hunk 10]
```diff
@@ -63,27 +63,25 @@ dependencies {
...
+    testImplementation "org.junit.jupiter:junit-jupiter-engine:5.7.0"
     testImplementation "org.junit.vintage:junit-vintage-engine:5.7.0"
     testImplementation "androidx.test:core:1.4.0-alpha04"
```
Comment (botbeaglezup) [inline android/beagle-scaffold/build.gradle:75]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold/build.gradle#L75"></span>A newer version of androidx.test:core than 1.4.0-alpha04 is available: 1.4.0-beta01</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/build.gradle [hunk 11]
```diff
@@ -63,27 +63,25 @@ dependencies {
...
     testImplementation "androidx.test:runner:1.4.0-alpha04"
```
Comment (botbeaglezup) [inline android/beagle-scaffold/build.gradle:76]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold/build.gradle#L76"></span>A newer version of androidx.test:runner than 1.4.0-alpha04 is available: 1.4.0-beta01</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/build.gradle [hunk 12]
```diff
@@ -63,27 +63,25 @@ dependencies {
...
+    testImplementation "junit:junit:4.13.2"
+    testImplementation "org.junit.jupiter:junit-jupiter-api:5.7.0"
+    testImplementation "io.mockk:mockk:1.10.2"
```
Comment (botbeaglezup) [inline android/beagle-scaffold/build.gradle:79]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold/build.gradle#L79"></span>A newer version of io.mockk:mockk than 1.10.2 is available: 1.10.6</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/build.gradle [hunk 13]
```diff
@@ -63,27 +63,25 @@ dependencies {
...
+
     androidTestImplementation "androidx.test.ext:junit:1.1.3-alpha04"
```
Comment (botbeaglezup) [inline android/beagle-scaffold/build.gradle:81]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold/build.gradle#L81"></span>A newer version of androidx.test.ext:junit than 1.1.3-alpha04 is available: 1.1.3-beta01</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          11 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults-sample/build.gradle#L57">android/beagle-defaults-sample/build.gradle#L57</a> - A newer version of androidx.core:core-ktx than 1.3.2 is available: 1.5.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults-sample/build.gradle#L58">android/beagle-defaults-sample/build.gradle#L58</a> - A newer version of androidx.appcompat:appcompat than 1.2.0 is available: 1.3.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults-sample/build.gradle#L67">android/beagle-defaults-sample/build.gradle#L67</a> - A newer version of org.jetbrains.kotlinx:kotlinx-coroutines-android than 1.3.9 is available: 1.4.2</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults-sample/build.gradle#L69">android/beagle-defaults-sample/build.gradle#L69</a> - Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (junit:junit:4.+)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults/build.gradle#L53">android/beagle-defaults/build.gradle#L53</a> - A newer version of androidx.core:core-ktx than 1.3.2 is available: 1.5.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-defaults/build.gradle#L54">android/beagle-defaults/build.gradle#L54</a> - A newer version of androidx.appcompat:appcompat than 1.2.0 is available: 1.3.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold-sample/build.gradle#L56">android/beagle-scaffold-sample/build.gradle#L56</a> - A newer version of androidx.core:core-ktx than 1.3.2 is available: 1.5.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold-sample/build.gradle#L57">android/beagle-scaffold-sample/build.gradle#L57</a> - A newer version of androidx.appcompat:appcompat than 1.2.0 is available: 1.3.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold-sample/build.gradle#L66">android/beagle-scaffold-sample/build.gradle#L66</a> - A newer version of org.jetbrains.kotlinx:kotlinx-coroutines-android than 1.3.9 is available: 1.4.2</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold-sample/build.gradle#L68">android/beagle-scaffold-sample/build.gradle#L68</a> - Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (junit:junit:4.+)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/8bd1a76a563682f5954878062da42e380e6a04a3/android/beagle-scaffold/build.gradle#L62">android/beagle-scaffold/build.gradle#L62</a> - A newer version of androidx.core:core-ktx than 1.3.2 is available: 1.5.0</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by uziasferreirazup — APPROVED** [review]

**Pull request merged**